foundations/javascript_basics/DOM_manipulation_and_events.md

```diff
@@ -55,7 +55,7 @@ So you're identifying a certain node based on its relationships to the nodes aro
 
 ### DOM methods
 
-When your HTML code is parsed by a web browser, it is converted to the DOM, as was mentioned above. One of the primary differences is that these nodes are JavaScript objects that have many properties and methods attached to them. These properties and methods are the primary tools we are going to use to manipulate our webpage with JavaScript. We'll start with the query selectors - those that help you target nodes.
```

Comment:
In what way is this sentence redundant?

Reply:
I made that change based on feedback from @thatblindgeye in #28030 but the gist of it is that the sentence 'We'll start with the query selectors - those that help you target nodes.' is unnecessary since the next section is titled 'Query Selectors' and explains what they do.

Reply:
Fair enough.